RegisterAndLogin/controller/AuthenticationController.go

```diff
@@ -0,0 +1,43 @@
+package controller
+
+import (
+	user "RegisterAndLogin/model"
+	"encoding/json"
+	"net/http"
+)
+
+type LoginDto struct {
+	Username string
+	Password string
+}
+
+var users = []user.User{
+	{Id: 1, Username: "Ranita", Password: "sifra123"},
+}
+
+func LogIn(w http.ResponseWriter, r *http.Request) {
```

Comment:
@NemanjaRanit Please provide basic documentation for this (and other) function(s).